content/privacy/gandalf.md

```diff
@@ -0,0 +1,5 @@
+---
+status: external
+---
+
+TODO: host Zeus dit nog?
```

Comment:
As far as I know enkel maar gehost door het FK

Reply:
No, still hosted on King. FK-Enrolment is hosted by the FK.